charts/ratify/README.md

```diff
@@ -49,6 +49,7 @@ Values marked `# DEPRECATED` in the `values.yaml` as well as **DEPRECATED** in t
 | tolerations                                        | Pod tolerations for the Ratify deployment                                                                                                                                                                                                                                                                                                                              | `[]`                              |
 | notationCerts                                      | An array of public certificate/certificate chain used to create inline certstore used by Notation verifier                                                                                                                                                                                                                                                             | ``                                |
 | cosignKeys                                         | An array of public keys used to create inline key management providers used by Cosign verifier                                                                                                                                                                                                                                                                         | `[]`                              |
+| notation.enabled                                   | Enables/disables notation signature lookup in ORAS store. MUST be set to true for notation verification.                                                                                                                                                                                                                                                               | `true`                            |
```

Comment:
Actually notation signature is a bit different from cosign signature. Even though notation.enable = true, Ratify will still look up notation signatures but just skip verification.
Therefore, I would say it just enables/disables the built-in Notation verifier.

Reply:
> Actually notation signature is a bit different from cosign signature. Even though notation.enable = true, Ratify will still look up notation signatures but just skip verification. Therefore, I would say it just enables/disables the built-in Notation verifier.

I assume you meant "Even though notation.enable = false" in your comment?

Reply:
you're correct, sorry for the confusion.